.github/workflows/performance.yml

```diff
@@ -9,10 +9,13 @@ on:
     tags:
       - '[6-9]+.[0-9]'
       - '[0-9]+.[0-9].[0-9]+'
+      - '![45].[0-9].[0-9]+'
+      - '!6.[01].[0-9]+'
   pull_request:
     branches:
       - trunk
-      - '6.[1-9]'
+      - 'feature/**'
```

Comment:
```suggestion
```

Once all the reviewer feedback has been addressed, we will remove this line.

Reply:
Feel free to remove this before merging this PR and make future updates directly in the feature branch. No need to get separate reviews on changes to the open PR on core.